pkg/controllers/dashboard/controller.go

```diff
@@ -47,6 +48,10 @@ func Register(ctx context.Context, wrangler *wrangler.Context, embedded bool, re
 
 	clusterconnected.Register(ctx, wrangler)
 
+	if features.UIExtension.Enabled() {
```

Comment:
This enablement of the controller is not needed now since you are starting the controller in all rancher pods now.